agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -53,7 +53,7 @@ const (
 
 	HostLogDir         = "/var/log/ecs/exec"
 	ContainerLogDir    = "/var/log/amazon/ssm"
-	ECSAgentExecLogDir = "/log/exec"
+	ECSAgentExecLogDir = "/var/log/ecs/exec"
```

Comment:
since this is same as `HostLogDir`, can the occurrences of `ECSAgentExecLogDir` be simply replaced as `HostLogDir` instead?